apps/vs-code-designer/src/app/commands/dotNetSDK/updateDotNetSDK.ts

```diff
@@ -0,0 +1,35 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { dotnet, dotNetPackageName, dotNetAptPackageName, dotNetSDKMajorVersion } from '../../../constants';
+import { ext } from '../../../extensionVariables';
+import { executeCommand } from '../../utils/funcCoreTools/cpUtils';
+import type { IActionContext } from '@microsoft/vscode-azext-utils';
+
+export async function updateDotNetSDK(context: IActionContext): Promise<void> {
+  ext.outputChannel.show();
+  context.telemetry.properties.isUpdating = 'true';
+  switch (process.platform) {
+    case 'win32':
+      await executeCommand(ext.outputChannel, undefined, 'winget', 'upgrade', `${dotNetPackageName}.${dotNetSDKMajorVersion}`);
+      break;
+    case 'linux':
+      await executeCommand(
+        ext.outputChannel,
+        undefined,
+        'sudo',
+        'apt-get',
+        'update',
+        '&&',
+        'sudo',
+        'apt-get',
+        'upgrade',
+        `${dotNetAptPackageName}-${dotNetSDKMajorVersion}.0`
+      );
+      break;
+    case 'darwin':
+      await executeCommand(ext.outputChannel, undefined, 'brew', 'install', `${dotnet}@${dotNetSDKMajorVersion}`);
```

Comment:
Same thing as the uninstall, do we need to check if it has brew and winget ?

Reply:
Per discussion - will work on this in a separate task to check package managers and help install if missing